Disco_Diffusion.ipynb

```diff
@@ -2718,7 +2720,7 @@
         "ExtraSetTop"
       ],
       "machine_shape": "hm",
-      "name": "Disco Diffusion v5.2 [w/ VR Mode]",
+      "name": "Disco Diffusion v5.1 [w/ Turbo]",
```

Comment:
This somehow reverted to the old text. Can you see what's happening with the converter and/or manually update it for now?